[Doc] Update example deploy doc for volumes (#6170)
skypilot-org/skypilot: docs/source/reference/api-server/examples/example-deploy-gke-nebius-okta.rst

* ``sky show-gpus`` to show available GPUs
* ``sky status`` to see SkyPilot status and infra available

> ✨ Bonus: Infiniband, Nebius shared filesystem, and PVC volumes

Reviewer: ```suggestion
✨ Bonus: Infiniband, Nebius shared filesystem, and Volumes
```

.. note::
   Add the above ``config`` field to the SkyPilot config (``~/.sky/config.yaml`` `global config <https://docs.skypilot.co/en/latest/reference/config.html#config-yaml>`_ or ``.sky.yaml`` `project config <https://docs.skypilot.co/en/latest/reference/config-sources.html#config-client-project-config>`_) to have the shared filesystem mounted automatically for all your jobs.

> PVC volumes support

Reviewer: ```suggestion
Volumes on top of Nebius shared filesystem
```

~~~~~~~~~~~~~~~~~~~

> SkyPilot supports creating and managing PVC (Persistent Volume Claim) volumes on Kubernetes clusters.

Reviewer: ```suggestion
The above guide allows you to mount the entire Nebius shared filesystem to all SkyPilot clusters. Additionally, SkyPilot supports creating and managing volumes on top of the shared filesystem using Kubernetes Persistent Volume Claims.

Volumes help isolate data and enforce storage limits per partition.
```

cc'ing @romilbhardwaj for accurate description.

Author: Updated `Volumes help isolate data and enforce storage limits per partition.` to `Volumes help isolate data per partition.` since Nebius fs CSI does not support storage limits in the same shared file system between PVCs yet.
